CHANGELOG.md

```diff
@@ -7,6 +7,7 @@
 ### New features
 
 * [#155](https://github.com/rubocop/rubocop-minitest/issues/155): Provide `assert_offense`, `assert_correction`, and `assert_no_offenses` testing APIs for custom Minitest cop development. ([@koic][])
+* [#152](https://github.com/rubocop/rubocop-minitest/issues/152): Add `ExtensionMatchers` configuration to `Minitest/GlobalExpectations`. ([@tejasbubane][])
```

Comment:
Can you use the following new changelog entry format instead?

> Added an entry (file) to the changelog folder named {change_type}_{change_description}.md if the new code introduces user-observable changes. See changelog entry format for details.

https://github.com/rubocop/rubocop-minitest/blob/master/.github/PULL_REQUEST_TEMPLATE.md

Reply:
Done